databases/backends/mysql.py

```diff
@@ -213,6 +212,16 @@ def _compile(
         logger.debug("Query: %s Args: %s", query_message, repr(args), extra=LOG_EXTRA)
         return compiled.string, args, CompilationContext(execution_context)
 
+    def _compile_many(
+        self, queries: typing.List[ClauseElement]
+    ) -> typing.Tuple[str, typing.Sequence[dict]]:
```

Comment:
```suggestion
    ) -> typing.Tuple[str, typing.List[dict]]:
```
IMO better be explicit in return value and as laxist as possible in args (Sequence, Mapping, ...)